### ISSUE TITLE
More granularity in the Permutor Cli iterations

### ISSUE THREAD

**Issue by WilliamsGomes1** [Proryanator/encoder-benchmark#20]
Is it possible to have a way to choose which settings the Permutor Cli will change in each iteration? For example, just iterate between the presets, while maintaining the same tune. It would save some time, mainly for the ones only interested in quality over latency/speed like me.

**Comment by Proryanator** [comment]
Yeah I could add in a parameter to pass in to "lock" or set a preset but permute over other things, that does sound nice.
